nixos/tests/grist-core.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test-python.nix (
```

Comment:
Migrate this to `runtTest` instead, see https://github.com/NixOS/nixpkgs/issues/38687.